COLLABORATOR_GUIDE.md

```diff
@@ -227,3 +228,44 @@ messages. However, you are only allowed to force push to any Node.js
 branch within 10 minutes from your original push. If someone else
 pushes to the branch or the 10 minute period passes, consider the
 commit final.
+
+### Long Term Support
+
+#### What is *LTS*?
+
+Long Term Support (often referred to as LTS) guarantee application developers 
+a 30 month support cycle with specific versions of Node.js.
+
+You can find more information [in the full LTS plan](https://github.com/nodejs/lts#lts-plan).
+
+#### How does *LTS* work?
+
+Once a release enters LTS, no new features may be added to that release. Changes are limited to
+bug fixes, security updates, possible npm updates, documentation updates, and certain
+performance improvements that can be demonstrated to not break existing applications, etc.
```

Comment:
Fixing it

Should we send a PR to https://github.com/nodejs/lts where I lifted the copy from?